pandas/core/indexes/multi.py

```diff
@@ -609,11 +609,28 @@ def _format_attrs(self):
         """
         Return a list of tuples of the (attr,formatted_value)
         """
+        def to_string_helper(obj, attr_name):
```

Comment:
instead of creating new functions, can you simply override ``_format_attrs`` or as appropriate

Reply:
ok so move this to pandas.io.formats.printing and make as generic as possible.